components/feature/webcompat-reporter/src/main/java/mozilla/components/feature/webcompat/reporter/WebCompatReporterFeature.kt

```diff
@@ -21,6 +23,14 @@ object WebCompatReporterFeature {
     internal const val WEBCOMPAT_REPORTER_EXTENSION_URL = "resource://android/assets/extensions/webcompat-reporter/"
     internal const val WEBCOMPAT_REPORTER_MESSAGING_ID = "mozacWebcompatReporter"
 
+    @VisibleForTesting
+    // This is an internal var to make it mutable for unit testing purposes only
+    internal var extensionController = WebExtensionController(
```

Comment:
The annotation above `@VisibleForTesting` make sure if someone tries to use this property outside of a testing context will get a compilation error :)

Reply:
I don't think I understand your comment, sorry. It was my intent that nobody except for us in tests can modify this, and I shamelessly copy-pasted the annotation, including the comment, [over from `feature-readerview`](https://github.com/mozilla-mobile/android-components/blob/cb73e4f9564dd1e888241a621843c9383fd7cd6e/components/feature/readerview/src/main/java/mozilla/components/feature/readerview/ReaderViewFeature.kt#L60-L61). Do you think that's a bad idea?

Reply:
Do not worry, the comment is OK :). My intention was just to let you know that when somebody sees the annotation already knows by annotation is self, that the property shouldn't be use outside of this context, the annotation itself is documenting the intention and enforcing it by a compilation check :)

Reply:
Ahhh, alright. Thanks for that clarification! :)

Reply:
My comment here is meant in addition to `VisibleForTesting`. It's not just visible (internal), it's also a var (not a val) for testing purposes. Maybe I should have written "This is an internal **var**` :)